balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -147,7 +157,6 @@ public Record(String entityId, String name, Number value, long timestamp, Metric
             while (true) {
                 Map<String, String> record = grovel(stream);
                 if (record == null) {
-                    log.error("Unable to process records from {}", filePath);
                     break;
```

Comment:
What does it mean when a null is returned from grovel? It's not (potentially) an error reading the file?